src/components/niivue-brain-grid.tsx

```
@@ -0,0 +1,675 @@
+                const diff = state.targetPosition - newPosition
+                newPosition += diff * 0.1
+                if (Math.abs(diff) < 0.01) {
+                  state.targetPosition = null
+                }
+              }
+            }
+            break
+            
+          case 'hunt':
+            // Hunting behavior - quick movements, then slow scan
+            const huntPhase = (time * 0.3) % 4
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)